backend/routes/fellows.js

```diff
@@ -60,93 +60,87 @@ const readFellowByEmail = async (req, res, next) => {
   }
 };
 
-// const createFellow = async (req, res, next) => {
-//   const firstname = processInput(req.body.f_first_name, res, "hardVarchar30", "firstname");
-//   const lastname = processInput(req.body.f_last_name, res, "hardVarchar30", "lastname");
-//   const picture = processInput(req.body.f_picture, res, "softVarcharNoLimit", "picture url");
-//   const bio = processInput(req.body.f_bio, res, "softVarcharNoLimit", "bio");
-//   const linkedIn = processInput(req.body.f_linkedin, res, "softVarchar150", "linkedin url");
-//   const github = processInput(req.body.f_github, res, "softVarchar150", "github url");
-//   const cohortId = processInput(req.body.cohort_id, res, "idNum", "cohort id");
-//   const wantMentor = processInput(req.body.want_mentor, res, "bool", "want mentor bool");
-
-//   try {
-//     const response = await queries.addFellow({
-//         firstname,
-//         lastname,
-//         picture,
-//         bio,
-//         linkedIn,
-//         github,
-//         cohortId,
-//         wantMentor
-//     });
-//     res.status(201);
-//     res.json({
-//         status: "success",
-//         message: `new fellow '${firstname} ${lastname}' added`,
-//         payload: response
-//     });
-//   } catch (err) {
-//     handleError(err, req, res, next);
-//   }
-// };
-
-// const updateFellow = async (req, res, next) => {
-//   try {
-//     const fellowId = processInput(req.params.fellow_id, res, "idNum", "fellow id");
-//     const firstname = processInput(req.body.f_first_name, res, "hardVarchar30", "firstname");
-//     const lastname = processInput(req.body.f_last_name, res, "hardVarchar30", "lastname");
-//     const picture = processInput(req.body.f_picture, res, "softVarcharNoLimit", "picture url");
-//     const bio = processInput(req.body.f_bio, res, "softVarcharNoLimit", "bio");
-//     const linkedIn = processInput(req.body.f_linkedin, res, "softVarchar150", "linkedin url");
-//     const github = processInput(req.body.f_github, res, "softVarchar150", "github url");
-//     const cohortId = processInput(req.body.cohort_id, res, "idNum", "cohort id");
-//     const wantMentor = processInput(req.body.want_mentor, res, "bool", "want mentor bool");
-
-//     const response = await queries.editFellow({
-//         fellowId,
-//         firstname,
-//         lastname,
-//         picture,
-//         bio,
-//         linkedIn,
-//         github,
-//         cohortId,
-//         wantMentor
-//     });
-//     res.status(201);
-//     res.json({
-//         status: "success",
-//         message: `fellow.${fellowId} data updated`,
-//         payload: response
-//     });
-//   } catch (err) {
-//     handleError(err, req, res, next);
-//   }
-// };
+const createFellow = async (req, res, next) => {
```

Comment:
No need to this function/route here 
This has beed addressed into routes/auth.js

Reply:
createFellow block in routes deactivated